packages/api/models/collections.js

```diff
@@ -52,32 +57,43 @@ class Collection extends Manager {
     return super.exists({ name, version });
   }
 
+  /**
+   * Creates a collection and puts it into a collection configuration store.
```

Comment:
I do have the ESLint plugin installed and working. By default, ESLint/JSDoc would not flag that as a JSDoc issue because JSDoc is based upon JavaDoc, and JavaDoc does not use that convention, nor is there any setting in `.eslintrc.json` that would require it.